force-app/main/default/lwc/geGatewaySelectWidget/__tests__/geGatewaySelectWidget.test.js

```diff
@@ -0,0 +1,298 @@
+            const element = createGeGatewaySelectWidget(null);
+            document.body.appendChild(element);
+            await flushPromises();
+
+            expect(getSpinner(element)).toBeFalsy();
+        });
+
+        it("displays payment methods, but not gateways when expanded", async () => {
+            GeGatewaySettings.setElevateSettings(null, null);
+            const element = createGeGatewaySelectWidget(null);
+            document.body.appendChild(element);
+            await flushPromises();
+
+            const expandPaymentControl = getExpandPaymentControl(element);
+            expect(expandPaymentControl).toBeTruthy();
+            expandPaymentControl.click();
+            await flushPromises();
+
+            expect(getComboBox(element)).toBeFalsy();
+            expect(getACHCheckBox(element)).toBeTruthy();
+            expect(getCCCheckBox(element)).toBeTruthy();
+        })
+
+        it("does not display payment methods when collapsed", async () => {
+            GeGatewaySettings.setElevateSettings(null, null);
+            const element = createGeGatewaySelectWidget(null);
+            document.body.appendChild(element);
+            await flushPromises();
+
+            const expandPaymentControl = getExpandPaymentControl(element);
+            expect(expandPaymentControl).toBeTruthy();
+            expandPaymentControl.click();
+            await flushPromises();
+
+            const collapseControl = shadowQuerySelector(element,'[data-id="pm-hide-button"]');
+            expect(collapseControl).toBeTruthy();
+            collapseControl.click();
+            await flushPromises();
+
+            expect(getComboBox(element)).toBeFalsy();
+            expect(getACHCheckBox(element)).toBeFalsy();
+            expect(getCCCheckBox(element)).toBeFalsy();
+        })
+    });
+
+    describe("gateway management mode", () => {
+        beforeEach(() => {
+            getGatewayAssignmentSettings.mockResolvedValue(JSON.stringify({
+                defaultGatewayId: DEFAULT_GATEWAY_ID,
+                defaultTemplateId: DEFAULT_TEMPLATE_ID,
+                gatewayAssignmentEnabled: true,
+            }));
+        });
+
+        it("renders widget contents when not default template", async () => {
+            GeGatewaySettings.setElevateSettings(null, SELECTED_GATEWAY_ID);
+            const element = createGeGatewaySelectWidget(GATEWAY_MANAGEMENT_MODE);
+            document.body.appendChild(element);
+            await flushPromises();
+
+            expect(element).toBeTruthy();
+            expect(getDiv(element)).toBeTruthy();
+        });
+
+        it("renders widget contents when default template", async () => {
+            GeGatewaySettings.setElevateSettings(null, DEFAULT_TEMPLATE_ID);
+            const element = createGeGatewaySelectWidget(GATEWAY_MANAGEMENT_MODE);
+            document.body.appendChild(element);
+            await flushPromises();
+
+            expect(element).toBeTruthy();
```

Comment:
What is this testing? Isn't Gateway Management Mode for the NPSP Settings page? The template has nothing to do with that, right?